tests/conftest.py

```diff
@@ -166,3 +167,13 @@ def mock_connection(mocker):
 def mock_successful_connection(mock_connection, successful_response):
     mock_connection.get.return_value = successful_response
     return mock_connection
+
+
+@pytest.fixture
+def mock_connection_post_failure(mocker, mock_connection):
```

Comment:
This fixture does not say enough about what it is... Should be

`mock_post_not_found_session` or something

Reply:
Renamed.